src/style.css

```diff
@@ -1,3 +1,76 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  background-color: #f2f2f2;
```

Comment:
this background doesn't required.
in design markup it is just default backround for figma